.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+extend-ignore = E203, E266, E501, W503, F403, F401, F841, C901
```

Comment:
oh wow, the flake list of code for this are pretty extensive: https://www.flake8rules.com/ 👍